### PR TITLE
Copied code

### PR DESCRIPTION
**Pull request by principlebrothers**
Design a cloned youtube page with HTML and CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>VidiTube</title>
+    <link href="stylesheet.css" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+    <section class="header-section">
+        <header>
```
Comment (principlebrothers) [inline index.html:13]: It will be semantically correct if you do not put a section over the header, main, and footer

File: index.html [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+                    </span>
+                    <span class="share">
+                        <img src="./images/share.svg" alt="share"/>
+                        Share
+                    </span>
+                    <span class="save">
+                        <img src="./images/save.svg" alt="save"/>
+                        Save
+                    </span>
+                </div>
```
Comment (principlebrothers) [inline index.html:66]: I think it would be more appropriate if you use a list for this

**Review by principlebrothers — COMMENTED** [review]
#Changes required ♻️ 

Please check my comments below and make all required changes.

## Please make sure you add linter to your folder

Submit for re-review when you are done.
Thank you 😁

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @principlebrothers 👋

Your project is complete! ✔️ 🏁. This is the First Project 1️⃣ You get Approved ✔️! in microverse 🌐.

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ good inline comments
- ✔️ good suggestive way

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by principlebrothers** [comment]
Hi @OmarRamoun, 
Thanks for the feedback. Your comments are soothing and inspiring.

I wish you saw the smile on my face when I read _"you have a great chance of becoming a great software engineer ⛑️ "_. 

Thank you 👍

**Pull request merged**